data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,325 @@
+				label end
+				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
+				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
+				action
+					set "incipias: helped the derelict"
+		personality
+			confusion 50
+			derelict uninterested staying
+	on enter "Hui'uc"
+		conversation
+			`On entering the system you find an alien spaceship disabled and badly damaged. There does not appear to be any other ships or inhabited planets in the system.`
+
+mission "Incipias: Help the stranded Part 1"
+	landing
+	source
+		government "Quarg (Incipias)"
+	destination "Pon'tes"
+	to offer
+		has "incipias: helped the derelict"
+	on offer
+		conversation
+			`Right after you arrived on the planet you go looking for a Quarg official to tell about the stranded ship. Luckily it is not long before you find one.`
+			`	"Human, you seem to be in quite a haste. What could be such a pressing matter?" it asks.`
+			choice
+				`	(Explain the situation.)`
+					goto explanation
+				`	"I need advice regarding the species in this region."`
+			`	It looks at you curiously. "So you met them? I am surprised that you were able to communicate. It took a very long time even for us to decipher their way of communication."`
+			`	You quickly explain what happened. The Quarg seems very relieved about the fact that you were not able to talk to them.`
+			label explanation
+			`	"You ran into a most peculiar situation. This species should not be able to cross the borders of their system yet." It tilts it head for a moment considering what you just said. "It is our desire to help them, but it is too early for us to reveal ourselves to their species. I fear there is nothing we can do." It shakes its head.`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"How can you make such a cold-hearted decision?"`
+			`	"This might be difficult for you to understand, but we must place the greater good above individual lives. If we show ourselves to them now, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
+			choice
+				`	"Sure, let me do the work."`
+				`	"If you are too weak..."`
+			`	Something like a laugh comes from the Quarg, before continuing.`
```

Comment:
If you mean for it to be the Quarg laughing, their equivalent of that appears to be clicking their teeth together rapidly. If you meant they make something akin to human laughter, which'd mean another reaction altogether, just ignore this comment.

Reply:
yea I meant the quarg to laugh, so I changed it to clicking